src/components/calcite-graph/calcite-graph.scss

```diff
@@ -7,3 +7,12 @@
   stroke: transparent;
   @apply m-0 p-0 block;
 }
+
+:host([data-style="slider-histogram"]) svg {
+  @apply w-full;
```

Comment:
Should this be baked into the graph or do we have use cases where we don't want the SVG to occupy the full width and the highlight color set?

Reply:
I wondered about this too. At first I added an optional `matchHostWidth` prop as a flag to graph for it to stretch the full width of its container if desired. (I went this route instead mainly because it seemed to be the way slider's histogram was originally styled.)

Reply:
@paulcpederson as the historical parent of `calcite-graph`, can you chime on this? 📈 👶🏼

Reply:
the width is always full in the slider, which was the original use case.

Reply:
Sorry for the delay, but is there a use case where the SVG wouldn't need to fill the whole width? If not, shouldn't it always be full width and up to the host component to modify its width?